python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml

```diff
@@ -0,0 +1,23 @@
+name: dining_prefs
+file_type: .csv
+author: J.L. Moreno
+url:  https://data.rapids.ai/cugraph/datasets/dining_prefs.csv
+refs:
+  J. L. Moreno (1960). The Sociometry Reader. The Free Press, Glencoe, Illinois, pg.35
+delim: " "
+header: None
+col_names:
+  - src
+  - dst
+  - wgt
+col_types:
+  - string
+  - string
+  - int
+has_loop: true
+is_directed: true
+is_multigraph: false
+is_symmetric: true
+number_of_edges: 52
+number_of_nodes: 26
+number_of_lines: 52
```

Comment:
The other YAML files no longer include `number_of_lines` in their metadata. Just # nodes and # edges